app/models.py

```diff
@@ -967,19 +967,28 @@ def clarification_questions_are_closed(self):
     def status(self):
         if self.published_at is None:
             return 'draft'
+        elif self.withdrawn_at:
```

Comment:
This is a "should never happen" scenario, but I think `withdrawn_at` should be checked first. If we consider a brief that for some reason has a `withdran_at` but no `published_at`:
* With current logic, brief status is set to draft, which means the brief can be published, at which point it will get a `published_at` and go straight to withdrawn.
* If we make `withdrawn_at` the first check then the brief is considered withdrawn. For a withdrawn brief `published_at` is mostly informational, so we're still missing some data, but I'd say the status is correct.